examples/docs_beta_snippets/docs_beta_snippets/guides/data-modeling/configuring-assets/config-schema.py

```diff
@@ -0,0 +1,29 @@
+import dagster as dg
+
+
+class ForecastModelConfig(dg.Config):
+    # parallelism defaults to 1, but can be overridden
+    # by the user. If you do not provide a default, the
+    # user will need to provide a value.
+    parallelism: int = 1
+
+
+@dg.asset
+def forecast_model(config: ForecastModelConfig):
+    print("Running with parallelism:", config.parallelism)  # noqa: T201
+    # ...more code here
```

Comment:
I think using `parallelism` might introduce unneeded confusion, as users may mistake this for concurrency of asset materialization. This might also be due to how we are not using the configuration parameter in our asset.

Can we consider using a different parameter that is set in a YAML file, and overridden by the config?

Reply:
Maybe something like `lookback_window: int = 60`